code/game/gamemodes/objective.dm

```diff
@@ -387,11 +387,12 @@ GLOBAL_LIST(admin_objective_list) //Prefilled admin assignable objective list
 	for(var/datum/mind/M in owners)
 		if(!M)
 			continue
-		var/datum/antagonist/changeling/changeling = M.has_antag_datum(/datum/antagonist/changeling)
-		if(!changeling)
```

Comment:
It looks like this was originally checking for if the target was NOT a changeling, but you have changed it to check for if it IS a changeling

Reply:
Good catch

Reply:
Wait no. I think this is right. This is doing the same thing, if the person has an antag datum (is changeling) then it skips them